### DISCUSSION TITLE
Is Codeproject nonfunctional?  I cannot reach server so the ai server doesn't work

### DISCUSSION THREAD

**Discussion by Armand29** [codeproject/CodeProject.AI-Server#139] [Q&A]
When I went into the server, the service was stopped and the "Install Modules" page was blank.  I installed 2.8.0 (I downloaded it last week, but the download site doesn't work today) and the page came up, told me I couldn't reach the server, then shut down.  Is codeproject.ai dead?

**Comment by ChrisMaunder** [comment]
It should work for now, but you will need to update the server soon. A new 2.9.1 is coming out this week
